assets/src/BlockEditorValidation.js

```diff
@@ -10,19 +10,22 @@ let canPublish = true;
 
 export const blockEditorValidation = () => {
   subscribe(() => {
-    const blocks = select( 'core/block-editor' ).getBlocks();
+    const blocks = select('core/block-editor').getBlocks();
 
     const currentMessages = [];
-    const invalidBlocks = blocks.reduce( (invalidBlocks, block ) => {
+    // eslint-disable-next-line no-shadow
+    const invalidBlocks = blocks.reduce((invalidBlocks, block) => {
       // Normally `blocks` contains a valid list of blocks, however it can happen that one of them is `null` in rare
       // cases. It happened to me once while running with WordPress 5.8.1 and undoing multiple edits. This made the
       // editor crash while it's trying to access `block.name`.
       if (!block) {
+        // eslint-disable-next-line array-callback-return
         return;
```

Comment:
Just thinking loud and not part of the scope.. In this case if you return the `accumulator` value will fix the `eslint` issue.

Reply:
I am not 100% sure, but I think this is like a hack to prevent the editor crashing. It is not to really return a value.